packages/webdriver-utils/test/providers/genericProvider.test.js

```diff
@@ -115,4 +121,30 @@ describe('GenericProvider', () => {
       });
     });
   });
+
+  describe('addPercyCSS', () => {
+    beforeEach(() => {
+      spyOn(Driver.prototype, 'executeScript').and.returnValue(Promise.resolve(true));
+    });
+
+    it('should call executeScript to add style', async () => {
+      genericProvider = new GenericProvider('123', 'http:executorUrl', { platform: 'win' }, {}, 'local-poc-poa', 'staging-poc-poa', {});
+      await genericProvider.createDriver();
+      await genericProvider.addPercyCSS('h1{color:green !important;}');
+      expect(genericProvider.driver.executeScript).toHaveBeenCalledTimes(1);
```

Comment:
Have expectation to have been called with and pass arguments as well. So in future if this method changed due to package update we fail the tests

Reply:
This function of `executeScript` has its own test and is being tested that way.